pkg/protocol/http1/resp/writer.go

```diff
@@ -0,0 +1,48 @@
+package resp
```

Comment:
filename  modify to "chunk_writer"?

Reply:
This file is suppose to hold all the extended writer impls(if any), so there is no need to expand them all under http1/resp dir - will be too trivial.